### ISSUE TITLE
sentry-sdk 1.37.0 tracebacks when eventlet is installed

### ISSUE THREAD

**Issue by nforro** [getsentry/sentry-python#2531]
### How do you use Sentry?

Self-hosted/on-premise

### Version

1.37.0

### Steps to Reproduce

Reproduced on CentOS Stream 9, version 1.36.0 is fine:

```bash
$ pip install eventlet
Successfully installed dnspython-2.4.2 eventlet-0.33.3 greenlet-3.0.1 six-1.16.0

$ pip install sentry-sdk
Successfully installed certifi-2023.11.17 sentry-sdk-1.37.0 urllib3-2.1.0

$ python3 -c "from sentry_sdk.integrations import Integration"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/__init__.py", line 1, in <module>
    from sentry_sdk.hub import Hub, init
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/hub.py", line 8, in <module>
    from sentry_sdk.scope import Scope
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/scope.py", line 7, in <module>
    from sentry_sdk.attachments import Attachment
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/attachments.py", line 5, in <module>
    from sentry_sdk.envelope import Item, PayloadRef
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/envelope.py", line 7, in <module>
    from sentry_sdk.session import Session
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/session.py", line 5, in <module>
    from sentry_sdk.utils import format_timestamp
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/utils.py", line 1241, in <module>
    HAS_REAL_CONTEXTVARS, ContextVar = _get_contextvars()
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/utils.py", line 1211, in _get_contextvars
    if not _is_contextvars_broken():
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/utils.py", line 1166, in _is_contextvars_broken
    greenlet_version = parse_version(greenlet.__version__)
NameError: name 'parse_version' is not defined
```

### Expected Result

No traceback.

### Actual Result

Traceback.

**Comment by sentrivana** [comment]
Fixed in `1.37.1` which will be out in a couple minutes -- thanks again for reporting!

**Comment by nforro** [comment]
Thank you for the fix!
